docs/src/pages/ui/theme/alternativeStyling/index.page.mdx

```diff
@@ -0,0 +1,501 @@
+---
+title: Alternative Styling
+---
+
+import Block from 'amplify-docs/src/components/Block';
+import BlockSwitcher from 'amplify-docs/src/components/BlockSwitcher';
+
+One of the goals of Amplify is to be a good citizen of the environment it exists in. As such Amplify works hard to not interfere with other tools that are being used.
+
+## CSS In JS
+
+There are many CSS In JS libraries out there that give a number of benefits. Amplify does not use a CSS In JS library to implement its theming system but operates well with the major CSS In JS libraries out there. Below are a few of the CSS In JS libraries out there and a description of how they interact with Amplify's theming system.
+
+<BlockSwitcher>
+<Block name="Styled Components">
+[Styled Components](https://github.com/styled-components/styled-components) allows you to create visual primitives with actual CSS code directly tied to components.
+
+#### Usage
+
+```jsx
+import React from 'react';
+import styled from 'styled-components';
+import { View } from '@aws-amplify/ui-react';
+
+const StyledView = styled(View)`
+  background-color: black;
+  font-size: 32px;
+`;
+
+//use it like any other amplify-ui component
+<StyledView color="red" className="my-styled-view">
+  This is my Styled View
+</StyledView>;
+```
+
+#### Interactions
+
+There are various ways to customize an Amplify component and Styled Components will interact with these customizations in the following ways:
+
+##### Amplify Styling Props
+
+These are styling props that can be used directly on an Amplify component which will affect a single style property. Examples of styling props are `color` or `fontWeight`. These Amplify styling props will take precedence over Styled Component styling.
+
+```jsx
+import React from 'react';
+import styled from 'styled-components';
+import { View } from '@aws-amplify/ui-react';
+
+const StyledView = styled(View)`
+  color: blue;
+`;
+
+<StyledView color="red">Using Styling props</StyledView>;
+```
+
+In the example above, the color of the view will be set to `red` because of the Amplify styling prop. [Here]() is a list of all Amplify styling props.
+
+##### Amplify variation props
+
+These are Amplify props that are added to certain Amplify components to change the look and/or behavior of the Amplify component. Examples of these variation props are `size` or `variation`.
```

Comment:
We could make this a bit tighter:

'These props change the look and/or behavior of certain Amplify components. Examples include `size` and `variation`.'